pttdb/ldb_database_test.go

```diff
@@ -1387,3 +1264,181 @@ func TestLDBBatch_TryPutAll(t *testing.T) {
 
 	// teardown test
 }
+
+func TestLDBBatch_TryPutAll2(t *testing.T) {
```

Comment:
why is this test called `TryPutAll2` if `TryPutAll` is already removed?

Reply:
TryPutAll is actually not removed.

The removed part is the duplicated code.